views/admin.hbs

```diff
@@ -1 +1,39 @@
 <h1 id="adminGreeting">Welcome, {{currentUser.firstName}} {{currentUser.lastName}}!</h1>
+<div class="row">
+  <div class="col s12">
+    <ul class="tabs">
+      <li class="tab col s3"><a href="#equipment">Equipment</a></li>
+    </ul>
+  </div>
+  <div id="equipment" class="col s12">{{> equipment }}</div>
+</div>
+
+<script src="/javascripts/materializeScriptInitializations.js"></script>
+<script type="text/javascript">
```

Comment:
Move to file

Reply:
Meh, it renders the data in the script (just the errors) and then runs the JS. Weirdness would need to be handled. Punting on this.